package.json

```diff
@@ -887,6 +887,11 @@
           "type": "boolean",
           "default": true
         },
+        "explorer.git.showOnlyGitChange": {
```

Comment:
I think we shouldn't need this configuration? Because the default use of `showOnlyGitChange` is too strange

Reply:
Yeah, I will remove it too. Just keep the action to toggle show only git changed files.

Reply:
@weirongxu Can you review it one more time.